### DISCUSSION TITLE
More update interval options.

### DISCUSSION THREAD

**Discussion by ETHREAL1** [microsoft/winget-cli#1587] [General] [answered]
Hello, I have been poking about with Winget and have noticed there is an Update interval setting. If you set it to 0 its disabled. But it would be nice if it could also be set to check for updates on system boot or shutdown. Say as -1 and -2 respectively or some such.

**Comment by denelon** [comment, marked as answer]
@ETHREAL1 the interval in settings is actually related to updating the "PreIndexed" package containing the local database of available packages. The Windows Package Manager does not have a service to automatically check for application upgrades.
